doc/source/faq.rst

```diff
@@ -58,22 +40,257 @@ For example, to set the limit to 4 GB of memory::
+has a separate log file located in :file:`/var/log/`.
+The file names have the following syntax: ``pmm-<type>-exporter-<port>.log``
+
+For example, the log file for the QAN monitoring service is
+:file:`/var/log/pmm-queries-exporter-42001.log`.
+
+You can view all available monitoring instance types and corresponding ports
+using the ``pmm-admin list`` command.
+For more information, see :ref:`pmm-admin-list`.
+
+.. _performance-issues:
+
+What are common performance considerations?
+===========================================
+
+If a MySQL server has a lot of schemas or tables,
+it is recommended to disable per table metrics when adding the instance:
+
+.. prompt:: bash
+
+   sudo pmm-admin add mysql --disable-per-table-stats
+
+If ``SELECT`` queries from ``information_schema`` tables slow down performace,
+you can disable all metrics from it when adding the instance:
+
+.. prompt:: bash
+
+   sudo pmm-admin add mysql --disable-infoschema
+
+For more information, run ``sudo pmm-admin add mysql --help``.
+
+Can I stop all services at once?
+================================
+
+Yes, you can use ``pmm-admin`` to start and stop either individual services
+that correspond to the added monitoring instances,
+or all of them at once.
+
+To stop all services:
+
+.. prompt:: bash
+
+   sudo pmm-admin stop --all
+
+To start all services:
+
+.. prompt:: bash
+
+   sudo pmm-admin start --all
+
+For more information about starting and stopping services,
+see :ref:`pmm-admin-start`.
+
+You can view all available monitoring instances
+and the states of the corresponding services
+using the ``pmm-admin list`` command.
+For more information, see :ref:`pmm-admin-list`.
+
+What privileges are required to monitor a MySQL instance?
+=========================================================
+
+When adding a :ref:`Query Analytics intance <pmm-admin-add-queries>`
+or a :ref:`MySQL metrics instance <pmm-admin-add-mysql>`,
+you can specify the MySQL server superuser account credentials,
+which has all privileges.
+However, monitoring with the superuser account is not secure.
+If you also specify the ``--create-user`` option,
+it will create a user with only the necessary privileges for collecting data.
+
+You can also set up the user manually with necessary privileges
+and pass its credentials when adding the instance.
+
+User for QAN monitoring
+-----------------------
+
+To add a local QAN instance,
+a command similar to the following is recommended:
+
+.. prompt:: bash
+
+   sudo pmm-admin add queries --user root --password root --create-user
+
+The superuser credentials are required only to set up the ``pmm-queries`` user
+with necessary privileges for collecting data.
+If you want to create this user yourself, the following privileges are required::
+
+ GRANT SELECT, PROCESS, SUPER ON *.* TO 'pmm-queries'@' localhost' IDENTIFIED BY 'pass' WITH MAX_USER_CONNECTIONS 5;
+ GRANT SELECT, UPDATE, DELETE, DROP ON performance_schema.* TO 'pmm-queries'@' localhost';
+
```

Comment:
Let's mention no `SUPER` needed when query source is `perfschema`. `--create-user` will handle this accordingly.